crates/tabby/src/routes/supported_models.rs

```diff
@@ -0,0 +1,48 @@
+use axum::Json;
+use serde::{Deserialize, Serialize};
+use tabby_common::config::Config;
+use utoipa::ToSchema;
+
+#[derive(Serialize, Deserialize, ToSchema, Clone, Debug)]
+pub struct SupportedModel {
+    completion: Vec<String>,
+    chat: Vec<String>,
+}
+
+#[utoipa::path(
+    get,
+    path = "/v1beta/models",
+    tag = "v1beta",
+    operation_id = "config",
+    responses(
+        (status = 200, description = "Success", body = ServerSetting, content_type = "application/json"),
+    ),
+    security(
+        ("token" = [])
+    )
+)]
+pub async fn models() -> Json<SupportedModel> {
+    let models: tabby_common::config::ModelConfigGroup =
+        Config::load().expect("Config file should be exist").model;
```

Comment:
Config shall be passed as a state from `serve.rs`.

See https://github.com/TabbyML/tabby/blob/80f34f9d8f7e1fac3cb27aa193456daed265347c/crates/tabby/src/routes/health.rs#L18 as an example (HealthState).